src/java/org/apache/cassandra/metrics/AccordMetrics.java

```diff
@@ -212,112 +255,128 @@ else if (txnId.isSomeRead())
         }
 
         @Override
-        public void onStable(Command cmd)
+        public void onStable(SafeCommandStore safeStore, Command cmd)
         {
+            Tracing.trace("Stable {} on {}", cmd.txnId(), safeStore.commandStore());
```

Comment:
did we ever get tracing to work with accord?  I recall it didn't work but don't recall us fixing it.